2.5/root/opt/app-root/etc/scl_enable

```diff
@@ -3,4 +3,4 @@
 #
 # This will make scl collection binaries work out of box.
 unset BASH_ENV PROMPT_COMMAND ENV
-source scl_source enable rh-ruby25 $NODEJS_SCL
+source scl_source enable $NODEJS_SCL rh-ruby25
```

Comment:
_Comment: Yes, this is the correct way. This order was in place from `2.4` at least, I recon it's not that much important as noone noticed._

Reply:
Exactly, there is probably no real impact, only if somebody installed npm into ruby's path and expected it override nodejs, which is not a real scenario.